clock/scheduledJobs/kueStatsActivityLogs.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * clock/scheduledJobs/kueStatsActivityLogs.js
+ *
+ * Kue stats activity logging.
+ */
+const key = 'kueStats';
+const k = require('../../config/activityLog').activityType[key];
+const warningThreshold = require('../../config').kueStatsInactiveWarning;
+const activityLogUtil = require('../../utils/activityLog');
+const jobQueue = require('../../jobQueue/setup').jobQueue;
+const DEFAULT_LOG_LEVEL = 'info';
+const WARNING_LOG_LEVEL = 'warn';
+const ZERO = 0;
+
+/**
+ * Generate the logging object. (Separating this out and exporting for easier
+ * testing.)
+ *
+ * @returns {Object} logging object
+ */
+function generateLogObject() {
+  const obj = JSON.parse(JSON.stringify(k));
+  jobQueue.card('active', (err, n) => {
+    if (!err) {
```

Comment:
the "inactiveCount" and "activeCount" etc. functions all call "card" ... their "card" function is exported, but I guess since it is not doc'd maybe it's considered not part of their public API... I'll change this to use the documented functions